reloj-checador/app/views/employees/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container mt-5 col-md-5">
+  <div class="card text-center">
+    <div class="card-header">
+      <%= @employee.name %>
+    </div>
+    <div class="card-body">
+      <span class="badge <%= @employee_presenter.badge_color %>"><%= @employee.status %></span>
+      <h5 class="card-title"><%= @employee.email %></h5>
+      <h5 class="card-title"><%= @employee.position %></h5>
+      <p class="card-text"><%= @employee.employee_number %></p>
+      <p class="card-text"><%= @employee.private_number %></p>
```

Comment:
It would be nice if they indicated what those values ​​mean, for example:

```suggestion
      <span class="badge <%= @employee_presenter.badge_color %>"><%= @employee.status %></span>
      <p class="card-text">Number:<%= @employee.employee_number %></p>
      <p class="card-text">Email: <%= @employee.email %></p>
      <p class="card-text">Position: <%= @employee.position %></p>
```

Remember that it should be the easiest thing to see for the end user.

Reply:
@YilKanda Thanks for the suggestion, you're right